docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"
```

Comment:
Is this the Python version to be used? If not, where is that defined?

Reply:
rtfm, sry.